src/client/delphi_epidata.py

```diff
@@ -74,12 +76,15 @@ def _request(params):
     """
     try:
       result = Epidata._request_with_retry(params)
-      if params is not None and "format" in params and params["format"]=="csv":
-        return result.text
-      else:
-        return result.json()
     except Exception as e:
       return {'result': 0, 'message': 'error: ' + str(e)}
+    if params is not None and "format" in params and params["format"]=="csv":
+      return result.text
+    else:
+      try:
+        return result.json()
+      except requests.exceptions.JSONDecodeError:
+        return {'result': 0, 'message': 'error: ' + result.text}
```

Comment:
nice condition handling!!
maybe make the message slightly more informative?
```suggestion
        return {'result': 0, 'message': 'error decoding json: ' + result.text}
```